oauthproxy.go

```diff
@@ -137,12 +137,20 @@ func NewOAuthProxy(opts *options.Options, validator func(string) bool) (*OAuthPr
 	}
 
 	logger.Printf("OAuthProxy configured for %s Client ID: %s", opts.GetProvider().Data().ProviderName, opts.ClientID)
+
 	refresh := "disabled"
-	if opts.Cookie.Refresh != time.Duration(0) {
-		refresh = fmt.Sprintf("after %s", opts.Cookie.Refresh)
+	cookieRefresh := opts.Cookie.Refresh
+	if !cookieRefresh.IsDisabled() {
+		if cookieRefresh.IsDurationBased() {
+			refresh = fmt.Sprintf("enabled, every %s", *cookieRefresh.RefreshDuration)
```

Comment:
```suggestion
			refresh = fmt.Sprintf("enabled, every %s seconds", *cookieRefresh.RefreshDuration)
```

Reply:
Nope. Duration of "7m" formats as "7m0s", what you'll get is "7m0s seconds".

> [2020/12/05 08:05:34] [oauthproxy.go:151] Cookie settings: name:_oauth2_proxy secure(https):false httponly:true expiry:1h0m0s domains: path:/ samesite: refresh:enabled, every 7m0s